src/Storages/System/InformationSchema/columns.sql

```diff
@@ -60,7 +60,7 @@ SELECT
     name AS column_name,
     position AS ordinal_position,
     default_expression AS column_default,
-    type LIKE 'Nullable(%)' AS is_nullable,
```

Comment:
I don't like either the modification in this PR or the original code.

Because it is doing ad-hoc, rough text parsing, like what you typically do with Perl.

The right way to do it is to introduce `is_nullable` column in the `system.columns` and reuse it here.